tests/e2e/specs/form-display-option-all-fields.test.js

```diff
@@ -1,294 +0,0 @@
-/**
```

Comment:
@henryholtgeerts can we easily repurpose these tests rather than deleting them?

Reply:
Just wrapped up an audit of the old tests this morning -- some of the specs are no longer relevant but what is still testing non-deprecated functionality I'll be using to inform new e2e tests (what to test, and how to test it). Because the old tests are not using the Cypress framework, the tests will need to be rewritten, but they do serve as useful guides/checklists for writing new tests. 👍